src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -169,7 +169,9 @@ class TowncrierDraftEntriesDirective(SphinxDirective):
 
     def run(self) -> List[nodes.Node]:  # noqa: WPS210
         """Generate a node tree in place of the directive."""
-        target_version = self.content[:1][0] if self.content[:1] else None
+        target_version = (  # noqa: WPS221
```

Comment:
Jones Complexity check only takes into account one line so by just structuring it in two lines there's no need to add noqa anymore.